Makefile

```
@@ -364,55 +408,95 @@ $(OBJECT_DIR)/$(TARGET)/%.o: %.S | $$(dir $$@)
+# the outer exit code (unlike prerequisite failure propagation in Make 4.4+).
+.PHONY: $(MAKECMDGOALS)
 
+$(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS)):
+	@:
+
+$(firstword $(MAKECMDGOALS)):
+	$(call summary_build,$(MAKECMDGOALS))
+
+else
+# Normal invocation (single target, named group, or inner sub-make).
 $(VALID_TARGETS):
```

checkmake (0.3.2)

[warning] Target body for "$(VALID_TARGETS)" exceeds allowed length of 5 lines (9).

(maxbodylength)